docs/reference/migration/migrate_7_2.asciidoc

```diff
@@ -9,6 +9,9 @@ your application to Elasticsearch 7.2.
 
 See also <<release-highlights>> and <<es-release-notes>>.
 
+* <<breaking_72_discovery_changes>>
+* <<breaking_80_ilm_deprecations>>
```

Comment:
Is this supposed to be breaking_80 or breaking_72?

Reply:
Should be `72`! Thanks for catching. Fixed with docs/reference/migration/migrate_7_2.asciidoc